src/lib/components/icon/icons/index.ts

```diff
@@ -6,6 +6,7 @@ export { default as Svg5Icon } from './5';
 export { default as Svg6Icon } from './6';
 export { default as Svg7Icon } from './7';
 export { default as Svg8Icon } from './8';
+export { default as CancerIcon } from './Cancer';
```

Comment:
Why did this move up here though?

Reply:
This file is auto generated so not sure why this happened 😅